pkg/image/apis/image/v1/helpers.go

```diff
@@ -0,0 +1,229 @@
+package v1
```

Comment:
what changes? as we move to versioned objects also in origin, the internal helpers will be deprecated :)